carta/cpp/CartaLib/ICoordinateFormatter.h

```diff
@@ -56,6 +56,9 @@ class CoordinateFormatterInterface {
     /// enable printing of an axis
     virtual Me & enableAxis(int ind) = 0;
 
+    /// Set the axes the user will see (3 axis: x,y axes plus a hidden axis).
+    virtual void setDisplayAxes( std::vector<Carta::Lib::AxisInfo::KnownType> displayAxes ) = 0;
+
```

Comment:
Does this have to be part of the interface class? I think the functionality could be entirely implemented already using just the public methods of this class.